rollup.config.mjs

```diff
@@ -1,13 +1,84 @@
 import zass from "./zass.mjs";
+import nodeResolve from "@rollup/plugin-node-resolve";
+import commonjs from "@rollup/plugin-commonjs";
+import babel from "@rollup/plugin-babel";
+import replace from "@rollup/plugin-replace";
+import { defineConfig } from "rollup";
 
-export default {
-  input: "src/index.js",
-  output: {
-    file: "script.js",
-    format: "iife",
+export default defineConfig([
+  {
+    input: "src/index.js",
+    output: {
+      file: "script.js",
+      format: "iife",
+    },
+    plugins: [zass()],
+    watch: {
+      clearScreen: false,
+    },
   },
-  plugins: [zass()],
-  watch: {
-    clearScreen: false,
+  {
+    input: ["src/modules/NewRequestForm.jsx", "src/modules/shared.jsx"],
```

Comment:
Would be nice to be consistent in terms of naming.
It doesn't seem like we're following the pattern of having the file name aligned with the default exported function name so I'm wondering if it makes sense to have some bundles in camel case and some other not.

Reply:
It makes sense to have consistent file naming. I think initially I just exported the component so I called the file with the component name, but now we export a function, so probably it makes sense to change the file name as well